lib/Importer/Caffe2.cpp

```diff
@@ -247,13 +247,14 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     }
 
     auto channel = getChannel(dict);
-    auto *node = G_.createBatchNormalization(opName, in, channel, epsilon);
-
-    // Load the weights.
-    cast<Variable>(node->getScale())->assign(scale);
-    cast<Variable>(node->getBias())->assign(bias);
-    cast<Variable>(node->getMean())->assign(mean);
-    cast<Variable>(node->getVar())->assign(var);
+    auto *scaleV = G_.getParent()->createVariable("scale", *scale);
```

Comment:
what is the plan for this?
(I thought it's to make these placeholders and then covertPlaceholderToConstants when appropriate)

Reply:
So I'm actually kind of unsure here.  We don't use Context at all in the loader right now, and plumbing it through to get at the placeholder versions of this (and FC and Conv) seems like kind of a lot of work.  So I'm conflicted.

Reply:
ok, we can take care of this at later point.

Reply:
I think that it makes sense to create constants (variables) for all of the weights that we load, and Placeholder for all of the uninitialized buffers (inputs/outputs). right?

Reply:
@nadavrot I think that makes sense if we are only going to use the proto loaders for pre-trained networks. If we eventually want to differentiate a graph that we load from a proto then I believe we would want to use Placeholders. We've received a question about this before (https://github.com/pytorch/glow/issues/1595).

Reply:
@jfix71 Yes, that's a good point, if we want to train from protos we'll need to create Placeholders here.  I think we can probably defer that for now, though.

Reply:
@bertmaher @jfix71 Ah, yes, I forgot about that. You are right.